ProjectLighthouse.Servers.Website/Pages/Errors/NotFoundPage.cshtml

```diff
@@ -4,8 +4,8 @@
 @{
     Layout = "Layouts/BaseLayout";
     Model.Title = "Not Found";
-    Model.Description = "The page was not found.";
+    Model.Description = "The requested page or resource was not found.";
 }
 
 <p>@Model.Description</p>
-<p>This may be due to a lack of permission such as not being signed in, or maybe the page just isn't there.</p>
+<p>You may need to <a href="/login">sign in</a> to access the requested page.</p>
```

Comment:
As in something like so?
```suggestion
<p><a href="/">Click here</a> to return to the homepage.</p>
```

Reply:
I was googling for 404 pages and something like this would probably work well.

![image2](https://help.piwik.pro/wp-content/uploads/2019/11/page-not-found.png)

This ones a bit more fancy but still is roughly the same thing
![image](https://user-images.githubusercontent.com/16675503/215311296-829c5bb3-5452-4fd4-acbc-b5e60a7443fe.png)